docs/src/user/benchmark.rst

```diff
@@ -27,6 +27,15 @@ Beside out of box :doc:`/code/benchmark/task` and :doc:`/code/benchmark/assessme
 you can also extend benchmark to add new ``Tasks`` with :doc:`/code/benchmark/task/base` and
 ``Assessments`` with :doc:`/code/benchmark/assessment/base`,
 
+To run benchmark with task :doc:`/code/benchmark/task/hpobench`, use ``pip install orion[hpobench]``
+to install the extra requirements. HPOBench provides local and containerized benchmarks, but different
+local benchmark could ask for total difference extra requirements. With ``pip install orion[hpobench]``,
+you will be able to run local benchmark ``benchmarks.ml.tabular_benchmark``.
+If you want to run other benchmarks in local, refer HPOBench `Run a Benchmark Locally`_. To run
+containerized benchmarks, you will need to install `singularity`_.
```

Comment:
Maybe there should be a quick word on how to use it with singularity once singularity is installed. Even if the usage is exactly the same, then it should be mentioned.